lib/mnemosyne/probes/seahorse/base.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Mnemosyne
+  module Probes
+    module Seahorse
+      module Base
+        class Probe < ::Mnemosyne::Probe
+          def setup
+            require 'mnemosyne/middleware/seahorse'
```

Comment:
Hmm, do we want to name it "handler" an place inside the probe directory?

I'm not a fan of the global middleware directory anymore...

Reply:
Yeah, makes sense. I named it "Plugin", as that's what we need to hook into Seahorse.